runtime/src/lib.rs

```diff
@@ -126,59 +86,73 @@ fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
     }
 
     // Get the command bytes
-    let packet = Packet::copy_from_mbox(drivers)?;
-    let cmd_bytes = packet.as_bytes()?;
+    let req_packet = Packet::copy_from_mbox(drivers)?;
+    let cmd_bytes = req_packet.as_bytes()?;
 
     cprintln!(
         "[rt] Received command=0x{:x}, len={}",
-        packet.cmd,
-        packet.len
+        req_packet.cmd,
+        req_packet.len
     );
 
-    match CommandId::from(packet.cmd) {
-        // FIRMWARE_LOAD expected to already be handled
-        CommandId::FIRMWARE_LOAD => Err(CaliptraError::RUNTIME_UNEXPECTED_UPDATE_RETURN),
+    // Some responses have a variable size
+    // Leave as None otherwise
+    let resp_size_ovrd: Option<usize> = None;
```

Comment:
Based on comment above, if the response contains the size, then the whole response can just be copied to the mailbox (at least for now)

Reply:
Changed this behavior and made a note in the code for now. Would be pretty easy go back to just sending what's needed later with the new size field in the response messages